services/outputhost/messagecache.go

```diff
@@ -979,12 +990,20 @@ func (msgCache *cgMsgCache) updateConn(connID int, event msgEvent) {
 
 func (msgCache *cgMsgCache) refreshCgConfig(oldOutstandingMessages int32) {
 	outstandingMsgs := oldOutstandingMessages
+	redeliveryIntervalInMs := msgCache.redeliveryIntervalInMs
+
 	cfg, err := msgCache.cgCache.getDynamicCgConfig()
 	if err == nil {
 		outstandingMsgs = msgCache.cgCache.getMessageCacheSize(cfg, oldOutstandingMessages)
+		redeliveryIntervalInMs = cfg.RedeliveryIntervalInMs
```

Comment:
ideally, this should made be a "per CG" config, like the "message cache size" config.